core/src/main/java/org/elasticsearch/index/query/MatchQueryBuilder.java

```diff
@@ -63,6 +63,10 @@
     /** The default name for the match query */
     public static final String NAME = "match";
 
+    //TODO all these alternate names are now deprecated but the MatchQueryBuilder code needs to be updated.
```

Comment:
this is for #17508 which I think should go in first. otherwise we deprecate before having an alternative.